src/ptr/CloRefString.zig

```diff
@@ -0,0 +1,122 @@
+//! Ref counted string type. The reference count is stored before the data.
+//! Zack said to name this string after myself; to leave my legacy.
```

Comment:
can we just name it RefString?

Reply:
ambiguous with `src/bun.js/jsc/RefString.zig`. what is ok is this isnt needed for this pr so i delete